themes/bootstrap3/templates/RecordDriver/DefaultRecord/controls/qrcode.phtml

```diff
@@ -0,0 +1,15 @@
+<?php if ($QRCode = $this->record($this->driver)->getQRCode($this->context)): ?>
+  <div class="dropdown hidden-xs">
+    <a href="#" class="dropdown-toggle qrcodeLink result-link icon-link" data-toggle="dropdown" aria-expanded="false" rel="nofollow">
+      <?=$this->icon('qrcode', 'icon-link__icon') ?>
+      <span class="icon-link__label"><?=$this->transEsc('qrcode_show')?></span>
+    </a>
+    <ul class="dropdown-menu qrcode">
+      <li>
+        <script type="text/template" class="qrCodeImgTag">
+          <img alt="" src="<?=$this->escapeHtmlAttr($QRCode);?>">
```

Comment:
Are you intentionally omitting the alt here? (I realize that an alt of "QR Code" isn't especially useful... but I wonder if it's better than nothing... also, we should be consistent across different QR Code displays, and the alt tag is still present in core.phtml and collection-info.phtml).

Reply:
It was intentional, but you're right in that it was also inconsistent. I restored the alt text here.